src/helpers/env.ts

```diff
@@ -17,6 +17,7 @@ export default cleanEnv(process.env, {
   BUNDLE_ID: str(),
   ETH_NETWORK: str({ default: ETH_MUMBAI_NETWORK }),
   ETH_RPC: str({ default: BWL_ETH_RPC }),
+  IPFS_GATEWAY: str({ default: 'https://ipfs.sealcred.xyz/ipfs' }),
```

Comment:
Hm, didn't we have this one in `bwl/env`?

Reply:
If we don't, we should add, I think

Reply:
no we don't have it